test/prompt/test_handlers.py

```diff
@@ -58,3 +58,21 @@ def test_flan_prompt_handler():
         "model_max_length": 20,
         "new_prompt_length": 10,
     }
+
```

Comment:
Just for me to understand, is there a reason why this entire test can't be refactored into 4 smaller unit test?

Reply:
To be fair I just copy and pasted them from the branch I mentioned in the PR description.

I wouldn't surprise me at all if they could be simplified. 😅